web/src/components/tasks/business-formation/business/IsVeteranNonprofit.tsx

```diff
@@ -0,0 +1,69 @@
+import { Content } from "@/components/Content";
+import { WithErrorBar } from "@/components/WithErrorBar";
+import { BusinessFormationContext } from "@/contexts/businessFormationContext";
+import { useConfig } from "@/lib/data-hooks/useConfig";
+import { useFormationErrors } from "@/lib/data-hooks/useFormationErrors";
+import { camelCaseToSentence } from "@/lib/utils/cases-helpers";
+import { FormControl, FormControlLabel, FormHelperText, Radio, RadioGroup } from "@mui/material";
+import { ReactElement, useContext } from "react";
+
+export const IsVeteranNonprofit = (): ReactElement => {
+  const { Config } = useConfig();
+  const { state, setFormationFormData } = useContext(BusinessFormationContext);
+  const fieldName = "isVeteranNonprofit";
+  const { doesFieldHaveError } = useFormationErrors();
+  const hasError = doesFieldHaveError(fieldName);
+
+  return (
+    <WithErrorBar hasError={hasError} type="ALWAYS">
+      <FormControl variant="outlined" fullWidth className="padding-bottom-2">
+        <label className="margin-right-3 text-bold">
+          <Content>{Config.formation.fields.isVeteranNonprofit.description}</Content>
+        </label>
+        <RadioGroup
+          aria-label={camelCaseToSentence(fieldName)}
+          className="fac"
+          key={`${state.formationFormData.isVeteranNonprofit}-key`}
```

Comment:
[pebble] We don't need this

Reply:
@jphechter: What is the "this" you are referring to? Can you be more specific?